tests/drivers/can/canfd/src/main.c

```diff
@@ -356,7 +356,7 @@ static void test_set_loopback(void)
 /**
  * @brief Test send/receive with standard (11-bit) CAN IDs and classic CAN frames.
  */
-void test_send_receive_classic(void)
+ZTEST(canfd_driver, test_send_receive_classic)
```

Comment:
```suggestion
ZTEST(canfd, test_send_receive_classic)
```

Please drop the `_driver` postfix from all test cases.